platform/pwl.c

```diff
@@ -1103,6 +1115,93 @@ zxdg_shell_v6_ping (void                 *data G_GNUC_UNUSED,
 #endif /* HAVE_XDG_SHELL_UNSTABLE_V6 */
 
 
+static PwlSeat*
+pwl_seat_create(struct wl_seat *seat, uint32_t name, uint32_t version, PwlDisplay *display)
```

Comment:
Add an space between the function name and the `(` parenthesis, and list each argument in its own line aligned to it. Also, I see that the function adds the seat to the `PwlDisplay` already, so a better name would be `display_add_seat()`, and there is no need to return the `PwlSeat`:

```c
static void
display_add_seat (PwlDisplay     *display,
                  struct wl_seat *seat,
                  uint32_t        name,
                  uint32_t        version)
```

Reply:
done